react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+function PokeList(props) {
+  const pokemon = props.pokedex;
+  const listPoke = pokemon.map((poke) => <li>{poke.name}</li>)
```

Comment:
You need to add a `key` prop to the `<li>` set to the `number` of the pokemon:

```JSX
<li key={poke.number}>{poke.name}</li>
```

Whenever you are doing a list react each item needs a `key` prop